js/viz/docs/docStatic.js

```diff
@@ -129,7 +138,7 @@ var staticMethods = {
     /**
     * @name vizmethods.getPalette
     * @publicName getPalette(paletteName)
-    * @param1 paletteName:string
+    * @param1 palette:string|Array<string>
     * @return object
```

Comment:
This method should return some Palette type, that contains all public API

Reply:
Must I describe new type `Palette` and return it instead of `object` ?

Reply:
Yes